json-form-editor/src/assets/js/main.js

```diff
@@ -28,6 +28,12 @@ contentfulExtension.init((extension) => {
 
   const fieldId = extension.field.id;
 
+  if (! json[fieldId]) {
+    const errorElement = document.createElement('h3');
```

Comment:
Not sure if this has to be an `h3` but not a big deal ;)

Reply:
LOL. Maybe another styling `@todo`?